static/gsApp/views/spikeProtection/spikeProtectionHistoryTable.tsx

```diff
@@ -288,14 +290,6 @@ const Title = styled('div')`
   gap: ${space(0.75)};
 `;
 
-const StyledCell = styled('div')`
-  display: flex;
-  align-items: center;
-  &:nth-child(5n) {
-    justify-content: end;
-  }
```

Comment:
Codemod should probably have caught this?

Reply:
Good catch! @cursoragent revert the changes here

Reply:
Oh no, wait. I remember what I did now. @natemoo-re I manually updated this to align the last cell to the end, see how it's using the justify end prop whereas the rest aren't using any